packages/devtools/lib/src/charts/flame_chart_canvas.dart

```diff
@@ -428,17 +367,19 @@ class FlameChartNode {
 
   final Color selectedTextColor;
 
-  final CpuStackFrame stackFrame;
+  final T data;
+
+  DisplayTextProvider displayTextProvider;
```

Comment:
Should this be a `DisplayTextProvider<T>` instead of a `DisplayTextProvider<dynamic>`

If we inline the function type, would you want `String Function(T)` or `String Function(dynamic)`?